usecase/pet.go

```diff
@@ -55,3 +59,16 @@ func (c *PetUseCase) ListUserPets(userID uniqueEntityId.ID) ([]*entity.Pet, erro
 	}
 	return pets, nil
 }
+
+func (c *PetUseCase) isValideSpecialCare(petToUpdate *entity.Pet) bool {
+	var needed = petToUpdate.NeedSpecialCare.Needed
+	var description = petToUpdate.NeedSpecialCare.Description
+
+	if needed {
+		return description != ""
+	}
+	if !needed {
+		return description == ""
+	}
+	return false
```

Comment:
Acredito que dá pra simplificar um pouco mais essa verificação, mas se achar confusa a sugestão pode deixar da forma que tá
```suggestion
	return (!needed && description == "") || (needed && description != "")
```

Reply:
Essa segunda eu havia apresentado ao @devlefel e ele achou que a primeira já resolvia o problema sem causar "pensamento a mais" para os mentorados

Reply:
E aí como nem reduzia quantidade de testes nem teria nenhum ganho além de ficar "aesthetic" ele me sugeriu corrigir para os 2 ifs

Reply:
pra mim, particularmente, foi um pouco mais difícil de entender mas só pelo fato de estar começando a me familiarizar como tudo funciona e como pode ser feito
porém, se for de acordo com vocês de aplicar essa última função eu faço a mudança

Reply:
Relaxa, pode deixar como tá com os ifs. Realmente não tem nenhum ganho real fazendo da forma que surgeri